rcp/org.testeditor.rcp4/src/org/testeditor/rcp4/dialogs/NetworkConnectionSettingDialog.java

```diff
@@ -359,46 +377,15 @@ protected void okPressed() {
 	 * @return if it is possible to reach internet resources.
 	 */
 	public boolean isInternetAvailable(boolean updateSessings) {
```

Comment:
Can we rename or refactor this a bit? The method is called `isInternetAvailable` but what it actually does is more like `checkIfIgnoreConnectionStateAndNotOfflineAndUpdateSystemSettingsAndCheckIfInternetAvailable` ;)